addons/hr/populate/hr.py

```diff
@@ -2,7 +2,7 @@
 
 from collections import defaultdict
 
-from odoo import models
+from odoo import models, fields
```

Comment:
alphabetical ordering:
fields, models

Reply:
Not sure alphabetical ordering is done anywhere; where would you put `_` in the ordering?

Reply:
https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html?highlight=guidelines#imports
at the end, according to the docs